src/calendar/day/basic/style.js

```diff
@@ -6,10 +6,15 @@ const STYLESHEET_ID = 'stylesheet.day.basic';
 export default function styleConstructor(theme={}) {
   const appStyle = {...defaultStyle, ...theme};
   return StyleSheet.create({
+    dayContainer: {
+      alignItems: 'center',
+      ...appStyle.dayContainerStyle,
+    },
     base: {
-      width: 32,
-      height: 32,
-      alignItems: 'center'
+      width: appStyle.dayBorderLength || 32,
+      height: appStyle.dayBorderLength || 32,
+      alignItems: 'center',
+      justifyContent: 'center'
```

Comment:
What is this `dayBorderLength` for?

Reply:
![image](https://user-images.githubusercontent.com/6138872/90207963-3b157780-de1a-11ea-87c3-8c2768d9d6e1.png)

In order to control the day selected style size, so get a side length.